### PR TITLE
Added the merged list and updated relevant vars

### PR DESCRIPTION
**Pull request by scpdavis1**
### Description

As requested in issue #3207:

- We included badwords-list npm package to update the existing profanities list.
- We then merged the existing list with the badwords-list package list. The merged list does not contain duplicates. 
- We deleted some profanities that were very short which could have caused false positives.

Closes #

### CODE REVIEW

File: backend/src/constants/profanities.ts [hunk 1]
```diff
@@ -1,420 +1,401 @@
...
-  "g00k",
-  "gayboy",
-  "gaygirl",
-  "gayz",
-  "god-damned",
-  "h00r",
-  "h0ar",
-  "h0re",
-  "hoar",
+exports.profanities = [
```
Comment (Bruception) [inline backend/src/constants/profanities.ts:4]: Please stick to project style and prefer declaration exports

Comment (scpdavis1) [reply backend/src/constants/profanities.ts:4]: OK I apologize for that mistake.

File: backend/src/constants/profanities.ts [hunk 2]
```diff
@@ -1,420 +1,401 @@
...
+   "xrated", 
+   "zabourah" 
 ];
 
-export const regexProfanities = profanities.map((profanity) => {
-  const normalizedProfanity = _.escapeRegExp(profanity.toLowerCase());
-  return `${normalizedProfanity}.*`;
-});
+
+var list = require('badwords-list'),
```
[the remark this answers is not in the document]
Comment (scpdavis1) [reply backend/src/constants/profanities.ts:388]: Alright I was not aware this was antiquated sorry about that. I have various other lists I can attempt that I think would work a lot better. I am going to test those. I appreciate the comments!

File: backend/src/constants/profanities.ts (hunk 1 above)
Comment (Bruception) [reply backend/src/constants/profanities.ts:4]: No harm done!

**Pull request closed without merging**